user_guides/jacc/pom.xml

```diff
@@ -207,21 +225,6 @@
                     </execution>
                 </executions>
             </plugin>
-	</plugins>
```

Comment:
@arjantijms Sorry for late review on this. 

I see a build failure for jacc userguide from the logs as below . Can you check if this is due to the changes in this file and resolve this if you see find it too.
cc @scottmarlow 

     [exec] [INFO] BUILD FAILURE
     [exec] [INFO] ------------------------------------------------------------------------
     [exec] [INFO] Total time: 19.409 s
     [exec] [INFO] Finished at: 2020-08-08T12:23:45Z
     [exec] [INFO] ------------------------------------------------------------------------
     [exec] [ERROR] Failed to execute goal org.asciidoctor:asciidoctor-maven-plugin:2.0.0-RC.1:process-asciidoc (generate-pdf-doc) on project tck_jacc: Execution generate-pdf-doc of goal org.asciidoctor:asciidoctor-maven-plugin:2.0.0-RC.1:process-asciidoc failed: (ENOENT) No such file or directory - file:/home/jenkins/agent/workspace/aee-tck-alwin_platformmoderename/user_guides/jacc/open-uri-cached-0.0.5.gemspec -> [Help 1]
     [exec] [ERROR]